### ISSUE TITLE
DataDecorator value is below minimum WCAG AA text contrast ratio on cg100 theme

### ISSUE THREAD

**Issue by ShayMurnin** [carbon-design-system/ibm-security#440]
The text contrast on the blue value in the `DataDecorator` component is flagging as a violation on the DAP tool (see screenshots).

> Contrast of 3.67 with its background is not sufficient to meet WCAG AA minimum requirements for text of size 14px and weight of 400.

Just for context – in our case on Falcon, we have a data table of QRadar offenses with each row containing at least one data decorator component to represent IP addresses relating to each offense. As a result, we're seeing this violation duplicated many times while running the DAP tool.

NOTE: I'm sure design might want to take a quick look but I had no luck when briefly playing around with the blue text theme tokens from Carbon to pass contrast on both light and dark theme. It seems the background color of the decorator ($hover-ui) doesn't play well with the blue text tokens. Changing the background to $field-01 resolved the contrast issue though so this could be a potential solution.

**Expected behavior -**
`DataDecorator` value should meet WCAG AA text contrast guidelines (minimum 4.5:1 ratio). It should not flag a DAP violation.

**Actual behavior -**
`DataDecorator` value does not meet WCAG AA text contrast guidelines. It flags a DAP violation.

### Steps for reproducing

1.  Go to the `DataDecorator` component's [storybook](https://ibm-security.carbondesignsystem.com/?path=/story/patterns-datadecorator--default)
2. Ensure theme is `Cool Gray 100`
3. Run the DAP tool in dev tools
4. View results

### Screenshots

![Screenshot 2020-03-13 at 10 45 07](https://user-images.githubusercontent.com/15728451/76616016-0889ea80-651b-11ea-8595-17b812cb08ef.png)
![Screenshot 2020-03-13 at 10 45 46](https://user-images.githubusercontent.com/15728451/76616020-0aec4480-651b-11ea-9a63-3a5afebbceeb.png)

**Comment by jendowns** [comment]
Thanks for the issue @ShayMurnin -- it looks like BOTH the gray100 and coolgray10 themes have violations? I"ll explore using a different token for either the background or the link color -- that may fix both. 🤔
